packages/docs/src/examples/va-card/Actions.vue

```diff
@@ -2,19 +2,31 @@
   <div class="row row">
     <div class="flex md6 lg4">
       <va-card>
-        <va-card-title>Between horizontal alignment</va-card-title>
+        <va-card-title>
+          {{ $t('api.VaCard.examples.actions.firstTitle') }}
+        </va-card-title>
         <va-card-actions align="between">
-          <va-button>Action 1</va-button>
```

Comment:
That's at best generic, or keep english.

Reply:
moved to generic